rules/data-preprocessing.smk

```diff
@@ -135,17 +135,38 @@ rule raw_land_cover:
 rule raw_protected_areas_zipped:
     message: "Download protected areas data as zip."
     output: protected("data/automatic/raw-wdpa.zip")
-    shell: "curl -sLo {output} -H 'Referer: {URL_PROTECTED_AREAS}' {URL_PROTECTED_AREAS}"
+    params:
+        url = URL_PROTECTED_AREAS.format(config["parameters"]["wdpa-date"])
+    shell: "curl -sLo {output} -H 'Referer: {params.url}' {params.url}"
 
 
 rule raw_protected_areas:
     message: "Extract protected areas data as zip."
     input: rules.raw_protected_areas_zipped.output
+    params:
+        year = config["parameters"]["wdpa-date"]
     output:
-        polygons = "build/raw-wdpa-feb2019/WDPA_Feb2019-shapefile-polygons.shp",
-        polygon_data = "build/raw-wdpa-feb2019/WDPA_Feb2019-shapefile-polygons.dbf",
-        points = "build/raw-wdpa-feb2019/WDPA_Feb2019-shapefile-points.shp"
-    shell: "unzip -o {input} -d build/raw-wdpa-feb2019"
+        polygons = "build/raw-wdpa/wdpa-shapes.shp",
+        polygon_data = "build/raw-wdpa/wdpa-shapes.dbf",
+        points = "build/raw-wdpa/wdpa-points.shp"
+    conda: "../envs/default.yaml"
+    shell:
+        """
+        set +e
+        unzip -o {input} -d build/raw-wdpa
+        unzip -o build/raw-wdpa/WDPA_{params.year}-shapefile0.zip -d build/raw-wdpa/WDPA_to_merge_0
```

Comment:
Can you run a `rm -r build/raw-wdpa/WDPA_to_merge_0` in the end, to delete the temporary files?

If shapefile wasn't such an ugly format, you could the easier and cleaner shadow directive of Snakemake. Unfortunately, it is.